install/build.py

```diff
@@ -604,6 +604,7 @@ def check_cub_version(compiler, settings):
 
     _cub_version = int(out)
     settings['define_macros'].append(('CUPY_CUB_VERSION_CODE', _cub_version))
+    settings['define_macros'].append(('THRUST_IGNORE_CUB_VERSION_CHECK', 1))
```

Comment:
The change of the line seems irrelevant to the PR.

Reply:
Removed! (maybe accidentally included...)